tools/build_langserver/langserver/hover_test.go

```diff
@@ -0,0 +1,65 @@
+package langserver
+
+import (
+	"context"
+	"core"
+	"path"
+	"testing"
+	"tools/build_langserver/lsp"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetHoverContent(t *testing.T) {
+	core.FindRepoRoot()
+	ctx := context.Background()
+	filepath := path.Join(core.RepoRoot, "tools/build_langserver/langserver/test_data/example.build")
+	uri := lsp.DocumentURI("file://" + filepath)
+
+	analyzer := newAnalyzer()
+	// Test hovering on the function call
+	content, err := getHoverContent(ctx, analyzer, uri, lsp.Position{Line: 0, Character: 3})
+	assert.Equal(t, nil, err)
+	expected := "def go_library(name:str, srcs:list, asm_srcs:list=None, hdrs:list=None, out:str=None, deps:list=[],\n" +
```

Comment:
be a bit careful with stuff like this in tests; this couples it quite closely to the specifics of the rule.